net/qpid-proton/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                qpid-proton
+description         Qpid Proton is a high-performance, lightweight AMQP \
+                    1.0 messaging library.
+long_description \
+    Qpid Proton is a high-performance, lightweight messaging library. It can \
+    be used in the widest range of messaging applications, including brokers, \
+    client libraries, routers, bridges, proxies, and more. Proton makes it \
+    trivial to integrate with the AMQP 1.0 ecosystem from any platform, \
+    environment, or language.
+categories          net
+license             Apache-2
+maintainers         {unifiedsoftworx.com:roddie.kieley @roddiekieley} openmaintainer
+platforms           darwin freebsd linux
+homepage            https://qpid.apache.org
+
+PortGroup           github 1.0
+
+github.setup        apache qpid-proton fa80534c87940b44fefd36abff4c5a9a79b99f47
```

Comment:
This is now ok, except that you need to override the version, else MacPorts will declare this port to be version `fa80...` and will refuse to update it later when the shasum will be "numerically lower".